Added CRC promo code (#347)
moderndive/ModernDive_book: index.Rmd

```{r, echo=FALSE, include=!knitr::is_latex_output(), purl=FALSE}
dev_version <- TRUE
```
<!-- include=FALSE for PDF sending to CRC -->
```{block, purl=FALSE, include=knitr::is_html_output()}

> This is the [website](https://moderndive.com/) for *Statistical Inference via Data Science:  A ModernDive into R and the Tidyverse*! Visit the [GitHub repository for this site](https://github.com/moderndive/ModernDive_book), find the book on [Amazon](https://www.amazon.com/Statistical-Inference-via-Data-Science/dp/0367409828/), or purchase it at [CRC Press](https://www.crcpress.com/Statistical-Inference-via-Data-Science-A-ModernDive-into-R-and-the-Tidyverse/Ismay-Kim/p/book/9780367409821) using promo code: **ASA18**.

Reviewer: ```suggestion
This is the [website](https://moderndive.com/) for *Statistical Inference via Data Science:  A ModernDive into R and the Tidyverse*! Visit the [GitHub repository for this site](https://github.com/moderndive/ModernDive_book), find the book on [Amazon](https://www.amazon.com/Statistical-Inference-via-Data-Science/dp/0367409828/), or purchase it at [CRC Press](https://www.crcpress.com/Statistical-Inference-via-Data-Science-A-ModernDive-into-R-and-the-Tidyverse/Ismay-Kim/p/book/9780367409821). Feel free to use promo code **ASA18** for a discount on the price.
```

Reviewer: @rudeboybert The sentence was getting a little long so I broke it into two sentences instead to make it a little clearer. Feel free to commit and merge if that looks OK to you.

> This is the [website](https://moderndive.com/) for *Statistical Inference via Data Science:  A ModernDive into R and the Tidyverse*! Visit the [GitHub repository for this site](https://github.com/moderndive/ModernDive_book), find the book on [Amazon](https://www.amazon.com/Statistical-Inference-via-Data-Science/dp/0367409828/), or purchase it at [CRC Press](https://www.crcpress.com/Statistical-Inference-via-Data-Science-A-ModernDive-into-R-and-the-Tidyverse/Ismay-Kim/p/book/9780367409821). Feel free to use promo code **ASA18** for a discount on the price.

Author: ```suggestion
This is the [website](https://moderndive.com/) for *Statistical Inference via Data Science:  A ModernDive into R and the Tidyverse*! Visit the [GitHub repository for this site](https://github.com/moderndive/ModernDive_book and find the book on [Amazon](https://www.amazon.com/Statistical-Inference-via-Data-Science/dp/0367409828/). You can also purchase it at [CRC Press](https://www.crcpress.com/Statistical-Inference-via-Data-Science-A-ModernDive-into-R-and-the-Tidyverse/Ismay-Kim/p/book/9780367409821) using promo code **ASA18** for a discounted price.
```

Author: @ismayc how about this instead? I think to minimize confusion we should put "CRC Press" and the discount code in the same sentence.
